verification/src/transaction_verifier.rs

```diff
@@ -822,3 +896,49 @@ impl<'a> OutputsDataVerifier<'a> {
         Ok(())
     }
 }
+
+/// Context-dependent checks exclude script
+///
+/// Contains:
+/// [`MaturityVerifier`](./struct.MaturityVerifier.html)
+/// [`SinceVerifier`](./struct.SinceVerifier.html)
+/// [`CapacityVerifier`](./struct.CapacityVerifier.html)
+/// [`FeeCalculator`](./struct.FeeCalculator.html)
+pub struct ContextualWithoutScriptTransactionVerifier<'a, DL> {
+    pub(crate) maturity: MaturityVerifier<'a>,
```

Comment:
Maybe we could use `TimeRelativeTransactionVerifier` instead of `SinceVerifier` and `MaturityVerifier`?